.ci.yaml

```diff
@@ -1071,7 +1071,11 @@ targets:
     properties:
       add_recipes_cq: "true"
       version_file: flutter_master.version
-      target_file: macos_check_podspecs.yaml
+      target_file: macos_repo_checks.yaml
+      dependencies: >
+        [
+          {"dependency": "swift_format", "version": "build_id:8797338979890974865"}
```

Comment:
The `version` tag is `version:2@505.chromium.2` which fails because the `@` isn't valid in the regex `^[a-z0-9_]+$` 
```
 ERROR: projects/flutter: cr-buildbucket.cfg: in "cr-buildbucket.cfg" (buckets #4 - try / swarming / builders #454 - Mac_x64 check_podspecs / caches #4): name: "packages_main_swift_format_version_2@505_chromium_2" does not match "^[a-z0-9_]+$"
```
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8758535881024973153/+/u/luci_validate/stdout

I don't think it's important which tag we use, `build_id` seems fine too.

Reply:
Actually it does matter because the tags are different on arm and x64.  I'll see if I can get the 3pp folks to push a shared version tag that matches the regex.